content/docs/concurrent-mode-suspense.md

```diff
@@ -83,91 +82,87 @@ function ProfileTimeline() {
-
-This demo is a teaser. Don't worry if it doesn't quite make sense yet. We'll talk more about how it works below. Keep in mind that Suspense is more of a *mechanism*, and particular APIs like `fetchProfileData()` or `resource.posts.read()` in the above example are not very important. If you're curious, you can find their definitions right in the [demo sandbox](https://codesandbox.io/s/frosty-hermann-bztrp).
-
-Suspense is not a data fetching library. It's a **mechanism for data fetching libraries** to communicate to React that *the data a component is reading is not ready yet*. React can then wait for it to be ready and update the UI. At Facebook, we use Relay and its [new Suspense integration](https://relay.dev/docs/en/experimental/step-by-step). We expect that other libraries like Apollo can provide similar integrations.
-
-In the long term, we intend Suspense to become the primary way to read asynchronous data from components -- no matter where that data is coming from.
-
-### What Suspense Is Not {#what-suspense-is-not}
+**[Essayez sur CodeSandbox](https://codesandbox.io/s/frosty-hermann-bztrp)**
 
-Suspense is significantly different from existing approaches to these problems, so reading about it for the first time often leads to misconceptions. Let's clarify the most common ones:
+Cette démo est là pour vous ouvrir l’appétit. Ne vous inquiétez pas si elle est déroutante à ce stade, nous la décrirons plus en détail dans un instant. Gardez à l’esprit que Suspense est davantage un *mécanisme*, et que les API spécifiques dans l’exemple ci-avant, telles que `fetchProfileData()` ou `resource.posts.read()`, n’ont que peu d’importance. Si vous êtes curieux·se, vous pouvez toujours en consulter l’implémentation directement dans la [sandbox de démonstration](https://codesandbox.io/s/frosty-hermann-bztrp).
 
- * **It is not a data fetching implementation.** It does not assume that you use GraphQL, REST, or any other particular data format, library, transport, or protocol.
+Suspense n’est pas une bibliothèque de chargement de données. C’est un **mécanisme à destination des bibliothèques de chargement de données** pour qu’elles puissent indiquer à React que *les données que lit un composant ne sont pas encore disponibles*. React peut alors attendre qu’elles le deviennent et mettre à jour l’interface utilisateur (UI). Chez Facebook, nous utilisons Relay et sa [nouvelle intégration avec Suspense](https://relay.dev/docs/en/experimental/step-by-step). Nous pensons que d’autres bibliothèques, telles qu’Apollo, fournirons des intégrations similaires.
 
- * **It is not a ready-to-use client.** You can't "replace" `fetch` or Relay with Suspense. But you can use a library that's integrated with Suspense (for example, [new Relay APIs](https://relay.dev/docs/en/experimental/api-reference)).
+Sur le long terme, nous comptons que Suspense devienne le moyen principal de lire des données asynchrone depuis des composants, et ce d’où que ces données viennent.
```

Comment:
J’aurais plutôt dit « nous voulons que » ou « nous souhaitons que » ?

Reply:
Ce n'est pas tant “we want” / “we wish” / “we intend” que “we expect”, mais le « nous nous attendons à ce que » est vraiment ampoulé. En général je simplifie en « nous pensons », mais là j'aimais pas.

« Nous estimons que… deviendra le moyen » ?

Reply:
« Nous prévoyons que... deviendra le moyen » peut être dans ce cas ? Sinon « estimons » me convient !